extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -513,6 +569,13 @@ export class DraggableDrawer extends AMP.BaseElement {
         this.mutateElement(() => resetStyles(this.element, ['transition']));
       }
 
+      if (
+        isPageAttachmentUiV2ExperimentOn(this.win) &&
+        this.element.tagName === 'AMP-STORY-PAGE-ATTACHMENT'
+      ) {
+        this.getPage_().classList.add('attachment-open');
+      }
```

Comment:
For this and following blocks:
A component should not perform any mutation outside of its own DOM tree. Please use the store service and the `ATTACHMENT_STATE` from the story-page.